bot.py

```diff
@@ -293,14 +304,30 @@ def get_oauth(client_id, client_secret, grant_type):
         "https://id.twitch.tv/oauth2/token?client_id=" + client_id + "&client_secret=" + client_secret + "&grant_type=" + grant_type)
     if oauth_request.status_code == 200:
         json_response = json.loads(oauth_request.text)
-        #print(json_response)
-        global oauth_timer
-        oauth_timer = (json_response['expires_in'])
         new_oauth_token = str((json_response['access_token']))
+    else:
+        print("Unable to obtain Twitch API OAUTH token")
+        new_oauth_token = ""
     return new_oauth_token
 
 
+def check_oath(oauth_token, client_id):
```

Comment:
Should this be check_oauth instead of check_oath?

Reply:
**oath**
_noun_
a solemn promise, often invoking a divine witness, regarding one's future action or behaviour.
"they took an oath of allegiance to the king"
....refactoring....

Reply:
problem solved my liege.